src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,77 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Calculator {
+
+    private static final String SEPARATOR = "[,:]";
+    private static final Pattern CUSTOM_SEPARATOR_PATTERN = Pattern.compile("//(.)\n(.*)");
+
+    public static int calculate(String text) {
+        if (isBlank(text)) {
+            return 0;
+        }
+        return sum(toInts(getSplit(text)));
+    }
+
+    private static String[] getSplit(String text) {
```

Comment:
```suggestion
    private static String[] split(String text) {
```
모든 메서드에서 get으로 시작하는 것보다 의미있는 동사로 시작하면 더 명확할 것 같아요 :)